packages/blade/src/components/BottomSheet/index.ts

```diff
@@ -0,0 +1,8 @@
+export {
```

Comment:
Also need to include the exports in root index

Reply:
Yes already did.